react-viewer/src/App.tsx

```diff
@@ -22,24 +29,33 @@ const App: React.FC = () => {
     );
   }
 
+  useEffect(() => {
+    const initOidc = async () => {
+      if (!MyTokenServerAuthClient.oidcClient) {
+        await MyTokenServerAuthClient.initializeOidc(tokenUrl);
+      }
+
+      try {
+        await MyTokenServerAuthClient.oidcClient.signIn();
+        setIsAuthorized(MyTokenServerAuthClient.oidcClient.isAuthorized);
+      } catch (error) {
+        // swallow the error. User can click the button to sign in
```

Comment:
dont you need to throw it here for the log on 45 to work?

Reply:
We'll be swallowing this error for `signin` whereas the line 45 will grab any error with the `initializeOidc`.